client/src/types/types.ts

```diff
@@ -53,6 +53,7 @@ interface SimulationParameter extends Donu.ModelParameter {
 type SimulationVariableValues = Array<{x: number, y: number}>;
 
 interface SimulationVariable extends Donu.ModelVariable {
+  edited?: boolean,
```

Comment:
Why do we need an edited on the Variables panel?

Reply:
We should also be able to add and remove variables from the model visualization to the Variables panel. Does this make sense @YohannParis ?

Reply:
Rosa explained that now Variables behaves like Parameters and you need to double-click on the model node to display them on the Variables panel.